Source/Core/DolphinWX/Debugger/CodeWindowFunctions.cpp

```diff
@@ -135,7 +135,7 @@ void CCodeWindow::OnProfilerMenu(wxCommandEvent& event)
 
     if (Core::GetState() == Core::State::Paused && PowerPC::GetMode() == PowerPC::CoreMode::JIT)
     {
-      if (g_jit != nullptr)
+      if (JitInterface::GetCore() != nullptr)
```

Comment:
Should we be giving out the core like that at all, if we can avoid it? Especially in this case where its only used to compare against `nullptr` (without keeping the result around to call something on it or pass it on further)...
Why is this check even here, isn't checking whether the core is paused or anything enough?